DataLoader/DataLoader/Program.cs

```diff
@@ -0,0 +1,125 @@
+﻿// This is a prototype tool that allows for extraction of data from an Azure Search index
+// Since this tool is still under development, it should not be used for production usage
+
+using Microsoft.Azure.Search;
+using Microsoft.Azure.Search.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AzureSearchBackupRestore
+{
+    class Program
+    {
+        private static string TargetSearchServiceName = [Target Search Service];
+        private static string TargetAPIKey = [Target Search Service API Key];
+        private static string TargetIndexName = [Target Index Name];
+
+        private static SearchServiceClient TargetSearchClient;
+        private static SearchIndexClient TargetIndexClient;
```

Comment:
Since you're using the REST API to upload the data, I don't think you need `SearchIndexClient`.